package.json

```diff
@@ -1,137 +1,135 @@
 {
-    "name": "@citizensadvice/design-system",
```

Comment:
Prettier autoformat, see https://github.com/citizensadvice/design-system/pull/246

Actual changes in here are to correct a couple of utility commands. View with [?w=1](https://github.com/citizensadvice/design-system/pull/247/files?w=1)